data/navy watcher subplot

```diff
@@ -0,0 +1,673 @@
+		"required reputation" 10
+		security 0.6
+		tribute 100
+			threshold 3500
+			fleet "Small Republic" 8
+
+event "lab on watcher"
+	system Algieba
+		trade Food 500
+		trade Clothing 354 
+		trade Metal 264
+		trade Plastic 454
+		trade Equipment 642
+		trade Medical 593
+		trade Industrial 649
+		trade Electronics 729
+		trade "Heavy Metals" 1065
+		trade "Luxury Goods" 1338
+	planet Watcher
+		attributes "dirt belt" research moon military
+		landscape land/hills4
+		description `Watcher is a small moon, the only world in this system that sustains any indigenous life. Because of the low gravity, there are forests here where the trees grow over two hundred meters tall.`
+		description `	The Navy has built a small surveillance outpost to deter poaching, and a small permanent settlement has begun sprouting around the new University of Hermes Frank Moore Research Laboratory.`
+		spaceport `The spaceport is little more than a few poured concrete slabs near the outpost and laboratory. A small general store acts as the tiny moon's marketplace.`
+		"required reputation" 10
+		security 0.6
+		tribute 300
+			threshold 3500
+			fleet "Small Republic" 8
+			fleet "Small Deep" 4
+
+event "knockout gas available"
+	outfitter "Lovelace Advanced"
+		"Knockout Gas"
+
+
+
+# FLEETS
+fleet "Drug Runners"
+	government "Pirate"
+	names "pirate"
+	cargo 3 5
+	personality
+		timid plunders
+	variant 5
+		"Argosy"
+	variant 3
+		"Argosy (Blaster)"
+	variant 2
+		"Argosy (Laser)"
+	variant 2
+		"Argosy (Missile)"
+	variant 2
+		"Argosy (Turret)"
+	variant 1
+		"Modified Argosy"
+	variant 1
+		"Modified Argosy (Heavy)"
+	variant 1
+		"Modified Argosy (Laser)"
+	variant 1
+		"Modified Argosy (Missile)"
+	variant 4
+		"Clipper"
+	variant 2
+		"Clipper (Heavy)"
+	variant 2
+		"Clipper (Speedy)"
+	variant 2
+		"Clipper"
+		"Hawk"
+	variant 1
+		"Clipper (Heavy)"
+		"Hawk (Rocket)"
+	variant 1
+		"Clipper (Speedy)"
+		"Hawk (Speedy)"
+	variant 3
+		"Argosy"
+		"Hawk"
+	variant 1
+		"Argosy (Blaster)"
+		"Hawk (Rocket)"
+	variant 1
+		"Argosy (Laser)"
+		"Hawk (Speedy)"
+	variant 1
+		"Argosy (Turret)"
+		"Hawk (Speedy)"
+	variant 1
+		"Argosy (Missile)"
+		"Hawk (Rocket)"
+	variant 1
+		"Modified Argosy"
+		"Sparrow" 2
+	variant 1
+		"Modified Argosy (Heavy)"
+		"Sparrow" 2
+	variant 1
+		"Modified Argosy (Laser)"
+		"Sparrow" 2
+	variant 1
+		"Modified Argosy (Missile)" 2
+	variant 8
+		"Bastion (Cargo)"
+	variant 3
+		"Modified Argosy (Heavy)" 2
+	variant 4
+		"Bastion (Cargo)"
+		"Modified Argosy (Heavy)" 2
+
+
+
+# JOBS
+mission "Researchers to Watcher"
+	name "Bring researchers to the lab on Watcher"
+	job
+	repeat
```

Comment:
I wanted it repeatable so that (a) the KO gas mission doesn't get offered until the player has brought a critical mass of researchers to the lab and (b) the player can continue bringing researchers to the lab after the side plot is over for a sort of a "living world" feeling - once you've helped establish the lab, it remains there and continues growing.